pkg/network/dns_stats_test.go

```diff
@@ -36,23 +36,23 @@ func testLatency(
 	var d = "abc.com"
 	sk := newDNSStatkeeper(DNSTimeoutSecs*time.Second, 10000)
 	key := getSampleDNSKey()
-	qPkt := dnsPacketInfo{transactionID: 1, pktType: Query, key: key, question: d}
+	qPkt := dnsPacketInfo{transactionID: 1, pktType: Query, key: key, question: d, queryType: DNSTypeA}
 	then := time.Now()
 	sk.ProcessPacketInfo(qPkt, then)
 	stats := sk.GetAndResetAllStats()
 	assert.NotContains(t, stats, key)
 
 	now := then.Add(delta)
-	rPkt := dnsPacketInfo{transactionID: 1, key: key, pktType: respType}
+	rPkt := dnsPacketInfo{transactionID: 1, key: key, pktType: respType, queryType: DNSTypeA}
 
 	sk.ProcessPacketInfo(rPkt, now)
 	stats = sk.GetAndResetAllStats()
 	require.Contains(t, stats, key)
 	require.Contains(t, stats[key], d)
 
-	assert.Equal(t, expectedSuccessLatency, stats[key][d].DNSSuccessLatencySum)
-	assert.Equal(t, expectedFailureLatency, stats[key][d].DNSFailureLatencySum)
-	assert.Equal(t, expectedTimeouts, stats[key][d].DNSTimeouts)
+	assert.Equal(t, expectedSuccessLatency, stats[key][d][DNSTypeA].DNSSuccessLatencySum)
+	assert.Equal(t, expectedFailureLatency, stats[key][d][DNSTypeA].DNSFailureLatencySum)
+	assert.Equal(t, expectedTimeouts, stats[key][d][DNSTypeA].DNSTimeouts)
```

Comment:
A general comment for all the tests: instead of using the same queryType everywhere, it might be good to mix them up (e.g., use `DNSTypeAAAA` for some cases).

Reply:
yes.  I was intending to go back and do additional tests with different record types later (separate PR).  Goal of this PR was to just get back to the same state; i.e. A record tests working as they did previously.